src/server/Privilege.ts

```
@@ -55,4 +58,101 @@ export class PrivilegeChecker {
+      const [layerKey, colorKey] = segment.split("-");
+      if (!layerKey || !colorKey) return "invalid";
+      const layer = this.cosmetics.flag.layers[layerKey];
+      const color = this.cosmetics.flag.color[colorKey];
+      if (!layer || !color) return "invalid";
+
+      const layerFlareOk =
+        layer.flares && flares && layer.flares.some((f) => flares.includes(f));
+      const colorFlareOk =
+        color.flares && flares && color.flares.some((f) => flares.includes(f));
+      const layerSuperFlareOk =
+        superFlare || (flares && flares.includes(`flag:layer:${layer.name}`));
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)